.github/workflows/test-turbo.yml

```diff
@@ -0,0 +1,189 @@
+name: Symfony UX Turbo
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    phpstan:
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.0'
+                  extensions: zip
+
+            - name: Get composer cache directory
+              id: composercache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composercache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+                  restore-keys: ${{ runner.os }}-composer-
+
+            - name: Install dependencies
+              working-directory: src/Turbo
+              run: composer install --prefer-dist
+
+            - name: Install PHPUnit dependencies
+              working-directory: src/Turbo
+              run: vendor/bin/simple-phpunit --version
+
+            - name: PHPStan
+              working-directory: src/Turbo
+              run: vendor/bin/phpstan analyse --no-progress
```

Comment:
👍 for psalm, also with https://github.com/psalm/psalm-plugin-symfony ( if the plugin is failing to detect type for something that is symfony related, let me know and i will try to get it fixed asap )

Reply:
Let's stick with phpstan for now, after merging i can migrate this ( and other package ) to psalm to be consistent with symfony framework ( people contributing would get used to one tool instead of two )